Shared/App/Lorem.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
I couldn't find a way around this. I needed to import UIKit in a bunch of places.